src/components/action.ts

```diff
@@ -7,29 +7,73 @@ import {
   property,
   TemplateResult,
 } from "lit-element";
-import { GameState } from "../common/types";
+import { GameState, Player } from "../common/types";
 
 interface BetTarget extends EventTarget {
   multiplier: number;
 }
 
+interface ActionTarget extends EventTarget {
+  action: string;
+}
+
 @customElement("action-element")
 export class Action extends LitElement {
   @property() public game!: GameState;
-  @property() private bet: string = "0";
+  @property() public player!: Player;
 
   protected render(): TemplateResult {
     return html`
-      <div class="action-box">
+      <div ?data-display=${this.player.isTurn} class="action-box">
         <div class="main-actions">
-          <button class="button red-button">Fold</button>
-          <button class="button green-button">Call</button>
-          <button class="button green-button">Check</button>
-          <button class="button blue-button">Raise</button>
-          <button class="button blue-button">Bet</button>
+          <button
+            .action=${"fold"}
+            @click=${this._callPlayerAction}
+            class="button red-button"
+            id="fold-button"
+            ?data-display=${this.game.currentBet > 0}
+          >
+            Fold
+          </button>
+          <button
+            .action=${"call"}
+            @click=${this._callPlayerAction}
+            class="button green-button"
+            id="call-button"
+            ?data-display=${this.game.currentBet > 0}
```

Comment:
We should use the same logic as the previous comment for changing the display of an element